neural_compressor/torch/algorithms/weight_only/gptq.py

```diff
@@ -30,9 +30,12 @@
 
 from neural_compressor.torch.utils import fetch_module, get_device, logger, set_module
 from neural_compressor.torch.utils.auto_accelerator import auto_detect_accelerator
+from neural_compressor.utils.utility import LazyImport

```

Comment:
Only `common` and `torch` folders are included in INC3[torch] binary. So `from neural_compressor.utils.xxx import xxx` is not allowed. Let me port the `LazyImport` to the `common/utils`.

Reply:
Ported `LazyImport` in https://github.com/intel/neural-compressor/pull/1701

Reply:
done